crt_portal/cts_forms/views.py

```diff
@@ -1068,27 +1068,114 @@ def post(self, request):
 
 class SavedSearchView(LoginRequiredMixin, FormView):
 
+    def get_section_args(self, section_filters):
+        section_args = ''
+        for section_filter in section_filters:
+            logging.info(section_filter)
+            section_args += f'&section_filter={section_filter}'
+        return section_args
+
     def get(self, request):
         saved_searches = SavedSearch.objects.filter().all()
-        selected_section = request.GET.get('section', None)
-        if selected_section:
-            saved_searches = SavedSearch.objects.filter(section=selected_section)
+        section_filter = request.GET.getlist('section_filter', [])
+        section_args = self.get_section_args(section_filter)
+        if section_filter:
+            saved_searches = SavedSearch.objects.filter(section__in=section_filter)
         output = {
+            'section_filter': section_args,
             'saved_searches': saved_searches,
+            'form': SavedSearchFilter(request.GET),
         }
         return render(request, 'forms/complaint_view/saved_searches/index.html', output)
 
     def post(self, request):
         saved_searches = SavedSearch.objects.filter().all()
-        selected_section = request.GET.get('section', None)
-        if selected_section:
-            saved_searches = SavedSearch.objects.filter(section=selected_section)
+        section_filter = request.GET.getlist('section_filter', [])
+        section_args = self.get_section_args(section_filter)
+        if section_filter:
+            saved_searches = SavedSearch.objects.filter(section__in=section_filter)
         output = {
+            'section_filter': section_args,
             'saved_searches': saved_searches,
+            'form': SavedSearchFilter(request.GET),
         }
         return render(request, 'forms/complaint_view/saved_searches/index.html', output)
 
 
+class SavedSearchActionView(LoginRequiredMixin, View):
+    form = SavedSearchActions
+
+    def get_section_args(self, section_filters):
+        section_args = ''
+        for section_filter in section_filters:
+            section_args += f'&section_filter={section_filter}'
+        return section_args
```

Comment:
Maybe make this a toplevel function, because it doesn't actually need `self` and is reused.